program/src/instruction.rs

```diff
@@ -146,6 +149,9 @@ pub enum PaladinGovernanceInstruction {
         /// The minimum required threshold of proposal rejection to terminate
         /// the proposal.
         proposal_rejection_threshold: u64,
+        /// The Paladin stake config account that this governance config account
+        /// corresponds to.
+        stake_config_address: Pubkey,
```

Comment:
Since the governance is a PDA based on the stake config address, it could be confusing to allow updating it. We should probably not allow changing it, unless there's a compelling reason.

Reply:
Ahhh I don't know why I didn't think of that tbh. Updated!